src/js/components/Peak/PeakDescription.tsx

```diff
@@ -24,13 +37,91 @@ const PeakDescription = ({ peak, showForm }: PeakDescriptionProps) => {
       <p>Województwo: {peak.region}</p>
       <p>Pasmo górskie: {peak.mountainRange}</p>
       <PeakStatistics peakId={peak.id} />
-      {showForm && (
-        <div className={styles.buttonBox}>
-          <SubmitButton
-            onClick={() => reachPeak(3000)}
-            text="Zaznacz jako zdobyty"
-            className={styles.reachPeakButton}
-          />
+      {true && (
+        <div>
+          <Popup
+            trigger={
+              <div className={styles.buttonBox}>
+                <SubmitButton
+                  text="Zaznacz jako zdobyty"
+                  className={styles.reachPeakButton}
+                />
+              </div>
```

Comment:
Ten przycisk można w sumie usunąć, skoro to jest zrobione jako modal. Jako trigger można ustawić tą ikonkę u góry `x Reach`.